test/cmdlineTests/standard_subassembly_offsets/output.json

```diff
@@ -0,0 +1,41 @@
+{
+    "contracts": {
+        "contract.sol": {
+            "Storage": {
+                "evm": {
+                    "bytecode": {
+                        "assemblyStructure": {
+                            "isCreation": true,
+                            "length": 1595,
+                            "start": 0,
+                            "subAssemblies": [
+                                {
+                                    "isCreation": false,
+                                    "length": 684,
+                                    "start": 877
+                                },
+                                {
+                                    "isCreation": true,
+                                    "length": 34,
+                                    "start": 1561,
+                                    "subAssemblies": [
+                                        {
+                                            "isCreation": false,
+                                            "length": 8,
+                                            "start": 26
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
```

Comment:
I noticed this while looking at the PR last time: the inner assembly is seemingly located within the code section of the outermost one. Is this a bug?

Reply:
Hmm... didn't we want the positions to be absolute in the end?

Reply:
We ultimately decided not to, as Sourcify was perfectly fine with this, and it would be a bit more annoying to do on our side; I even documented it as is. Daniel was also a proponent of relative offsets.